nifi-nar-bundles/nifi-graph-bundle/nifi-other-graph-services/src/test/java/org/apache/nifi/graph/GremlinClientServiceIT.java

```diff
@@ -42,20 +42,20 @@ public void setup() throws Exception {
         clientService = new TestableGremlinClientService();
         runner = TestRunners.newTestRunner(NoOpProcessor.class);
         runner.addControllerService("gremlinService", clientService);
-        runner.setProperty(clientService, AbstractTinkerpopClientService.CONTACT_POINTS, "localhost");
```

Comment:
This class should be renamed and the tests expanded a bit to flex the functionality.

Reply:
Tracking, I am planning on building out integration tests utilizing test containers, will be doing that shortly after I verify the most recent changes work with my local test env.